source/system/structs/HashMap.ooc

```diff
@@ -95,262 +60,149 @@ acX31Hash: func <K> (key: K) -> SizeT {
 	h
 }
 
-getStandardHashFunc: func <T> (T: Class) -> Func <T> (T) -> SizeT {
-	if (T == String)
-		acX31Hash
-	else if (T size == Pointer size)
-		pointerHash
-	else if (T size == UInt size)
-		intHash
-	else
-		murmurHash
-}
-
 HashMap: class <K, V> {
-	_count: SizeT
-	capacity: SizeT
-	hashKey: Func <K> (K) -> SizeT
-	buckets: HashEntry[]
-	keys: VectorList<K>
```

Comment:
I don't think so. You can search our codebase and see what's the most common use case. If people call `hash keys()` a lot, then it's probably smart to optimize it somehow. Otherwise I think your implementation is ok.